qcodes/logger/logger.py

```diff
@@ -145,6 +151,17 @@ def get_log_file_name() -> str:
                         PYTHON_LOG_NAME)
 
 
+
+def flush_telemetry_traces() -> None:
+    """
+    Flush the traces of the telemetry logger. If telemetry is not enabled, this
+    function does nothing.
+    """
+    if qc.config.telemetry.enabled:
+        global telemetry_handler
+        telemetry_handler.flush()
```

Comment:
Does it make sense to hook this function up to atexit?

Reply:
Good question. The `logging.shutdown()` already flushes, so it might be superfluous. I'll try to dig into that.